packages/test-helpers/bin/writeComponentsTests.js

```diff
@@ -1,28 +1,67 @@
 #!/usr/bin/env node
-
+const program = require("commander");
 const fs = require("fs");
 
+const version = require("../../../package").version;
+
+program
+  .version(version)
+  .option(
+    "-d, --directory <directory>",
+    "Specify fructose root, default is [.fructose]"
+  )
+  .option(
+    "-t, --test-entry-filename <testEntryFile>",
+    "Specify the test entry file name, default is [components.test.js]"
+  )
+  .option(
+    "-a, --app-entry-filename <appEntryFile>",
+    "Specify the app entry file name, default is [components.js]"
+  )
+  .parse(process.argv);
+
+const directory = program.directory ? program.directory : ".fructose";
+const testEntryFilename = program.testEntryFilename
+  ? program.testEntryFilename
+  : "components.test.js";
+const appEntryFilename = program.appEntryFilename
+  ? program.appEntryFilename
+  : "components.js";
+
+if (!directory) {
+  throw new Error("fructose root not specified, specify with -d");
+}
+
+if (!testEntryFilename) {
+  throw new Error("test entry file not specified, specify with -t");
+}
+
+if (!appEntryFilename) {
+  throw new Error("app entry file is not specified, specify with -a");
+}
+
 const writeFile = (data, path) =>
   new Promise((resolve, reject) => {
     fs.writeFile(path, data, err => {
       if (err) {
         reject(err);
       }
-      resolve("wrote tests to file");
+      console.log("wrote tests to file");
```

Comment:
I think our lint rules don't like `console.log`

Reply:
should be fine here, they don't like it but it only pops up as a warning